src/calculateRentalCost.js

```diff
@@ -27,8 +27,24 @@
  * @return {number}
  */
 
+const PRICE = 40;
+const SMALL_DISCOUNT = 20;
+const BIG_DISCOUNT = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  const TOTAL_PRICE = PRICE * days;
```

Comment:
Hey, this value should in camelCase 🙂
```suggestion
  const totalPrice = PRICE * days;
```

Reply:
Thank you very much for a link!